platform/broadcom/sonic-platform-modules-cel/debian/platform-modules-silverstone.install

```diff
@@ -0,0 +1,6 @@
+silverstone/scripts/sensors usr/bin
+silverstone/scripts/platform_sensors.py usr/local/bin
+silverstone/cfg/silverstone-modules.conf etc/modules-load.d
+silverstone/systemd/platform-modules-silverstone.service lib/systemd/system
+silverstone/modules/sonic_platform-1.0-py2-none-any.whl usr/share/sonic/device/x86_64-cel_silverstone-r0
```

Comment:
why this is required?

Reply:
to support reboot cause service
https://github.com/Azure/sonic-buildimage/blob/201811/files/image_config/process-reboot-cause/process-reboot-cause

ref : https://github.com/Azure/sonic-buildimage/pull/3198